components/common/TakeAction.js

```diff
@@ -1,30 +1,39 @@
 import React from 'react';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { faArrowRight } from '@fortawesome/free-solid-svg-icons';
+import { useRouter } from 'next/router'
 
 import styles from './Common.module.css';
 
+const links = [
+    {path: "/contact-your-legislators", label: "Contact Your Legislators"},
```

Comment:
Looks like this page currently exists as `contact-legislators` not `contact-your-legislators`